app/src/main/res/values/dimens.xml

```diff
@@ -27,6 +30,7 @@
     <dimen name="saving_accounts_text_view_margin_left">50dp</dimen>
     <dimen name="loan_accounts_text_view_margin_left">50dp</dimen>
     <dimen name="client_charge_text_view_margin_left">20dp</dimen>
+    <dimen name="row_status_margin_end">8dp</dimen>
```

Comment:
According to android Material Guideline, 8dp is the standard margin.
So, ```row_status_margin_end``` can be ```default_margin``` and also remove 
``` <dimen name="login_text_input_margin_top">8dp</dimen>
    <dimen name="login_text_input_margin_bottom">8dp</dimen>
```
and use ```default_margin```.